src/uct/api/uct.h

```diff
@@ -1303,27 +1385,19 @@ ucs_status_t uct_iface_reject(uct_iface_h iface,
  * @ingroup UCT_RESOURCE
  * @brief Create new endpoint.
  *
- * @param [in]  iface   Interface to create the endpoint on.
+ * @param [in]  params  User defined @ref uct_ep_params_t configurations for the
+ *                      @a ep_p.
  * @param [out] ep_p    Filled with handle to the new endpoint.
- */
-ucs_status_t uct_ep_create(uct_iface_h iface, uct_ep_h *ep_p);
-
-
-/**
- * @ingroup UCT_RESOURCE
- * @brief Create an endpoint which is connected to remote interface.
  *
- * requires @ref UCT_IFACE_FLAG_CONNECT_TO_IFACE capability.
- *
- * @param [in]  iface       Interface to create the endpoint on.
- * @param [in]  dev_addr    Remote device address to connect to.
- * @param [in]  iface_addr  Remote interface address to connect to.
- * @param [out] ep_p        Filled with handle to the new endpoint.
+ * @return UCS_OK       The endpoint is created successfully. This does not
```

Comment:
I would add to doc the 3 connection modes:
1. no address -> need to use ep_connect_to_ep later, requires iface flag ...
2. dev_addr+iface_addr -> connect to remote interface, requires iface flag ...
3. sockaddr -> connect to remote server, requires iface flag ...

Reply:
Added

Reply:
field_mask: would -> will